common-libs/url-resolver/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# release 1.1.3 (2022-04-04)
+Remove unused service 
+  - user auth 
+  - sff-issuer 
```

Comment:
Can we keep it to a separate MR?

Reply:
nop tests are not running any more

Reply:
But they did just a week ago, did something change?